index.html

```diff
@@ -2612,6 +2617,96 @@ <h3 id="applying">
       </section>
       <section>
         <h3>
+          <code title="">category</code> member
+        </h3>
+        <p>
+          The <dfn id="member-category"><code>category</code>
+          member</dfn> describes the expected application category to which the 
+          web application belongs. This is a hint to catalogs or stores of web 
+          applications under which category (or categories) to list this web 
+          application.
+        </p>
+        <p>
+          The <code>category</code> member is only meant as a hint to catalogs 
+          or stores listing web applications and it is expected that these 
+          will make a best effort to find an appropriate category under which 
+          to list the web application.
+        </p>
+        <p>
+          The <dfn>steps for processing the <code>category</code>
+          member</dfn> are given by the following algorithm. The algorithm
+          takes a <var>manifest</var> as an argument. This algorithm returns a
+          string or <code>undefined</code>. The string MAY include a comma 
+          delimited list of categories.
+        </p>
+        <ol>
+          <li>Let <var>value</var> be the result of calling the
+          <a>[[\GetOwnProperty]]</a> internal method of <var>manifest</var>
+          with argument "<code>category</code>".
+          </li>
+          <li>If <a>Type</a>(<var>value</var>) is not "string":
+            <ol>
+              <li>If <a>Type</a>(<var>value</var>) is not "undefined", <a>issue
+              a developer warning</a> that the type is not supported.
+              </li>
+              <li>Return <code>undefined</code>.
+              </li>
+            </ol>
+          </li>
+          <li>Return <var>category</var>.
+          </li>
+        </ol>
+        <p>
+          The category string should be treated as case insensitive. Thus, 
+          <code>sports</code>, <code>Sports</code>, <code>SPORTS</code>, and 
+          <code>SpOrTs</code> are all equivalent.
+        </p>
+        <p>
+          Spaces, underscores, and dashes should be treated as equivalent. 
+          Thus, <code>health & fitness</code>,  
+          <code>health_&_fitness</code>, and <code>health-&-fitness</code> 
+          are all equivalent.
+        </p>
+        <p>
+          If a category string has multiple comma delimited values, a 
+          consuming system SHOULD use the most specific value that is is 
+          familiar with, and ignore values that it is not familiar with.  
+          For example: <code>Books, Fiction, Science Fiction & Fantasty</code>.
+        </p>
+        <p>
+          Manifest authors SHOULD use the below common categories:
+          <ul>
+            <li>books</li>
+            <li>business</li>
+            <li>education</li>
+            <li>entertainment</li>
+            <li>finance</li>
+            <li>food</li>
+            <li>games</li>
+            <li>government & politics</li>
```

Comment:
I don't think this list should be in the spec. We should move it somewhere else where it can be updated and maintained independently of the spec.